src/controller/admin.py

```diff
@@ -14,7 +14,7 @@
 login_manager = Shared.login_manager
 
 
-@admin.route('/api/admin/reload', methods=['POST'])
+@admin.route('/api/admin/reload', methods=['POST', 'GET'])
```

Comment:
This makes it easier for testing, but we should remove it when we get a working file upload solution.

Reply:
I think I committed this file by accident